src/_data/defaultSchedule.yml

```diff
@@ -1,61 +1,56 @@
 monday:
   - start: 9:45
-    end: 10:00
-    name: Check-in
-  - start: 10:00
-    end: 10:10
-    name: Feedback survey
-  - start: 13:00
-    end: 14:00
-    name: Lunch
-  - start: 16:00
-    end: 16:15
-    name: Project & Spikes intro
-    url: ../project
-  - start: 16:15
-    end: 17:15
-    name: Technical spikes
-    type: spike
-    url: ../spikes
-  - start: 17:15
-    end: 17:45
-    name: Spike presentation preparation
-    url: /course/handbook/spikes/#what-makes-a-good-research-presentation
-  - start: 17:45
     end: 18:00
-    name: Check-out
+    name: Workshops
+    type: workshop
 tuesday:
   - start: 9:45
     end: 10:00
     name: Check-in
   - start: 10:00
-    end: 10:15
-    name: Spike prep
-    url: /course/handbook/spikes/#what-makes-a-good-research-presentation
-  - start: 10:15
+    end: 10:10
+    name: Feedback survey
```

Comment:
It feels a little weird to have this here. Could we not just ask them to do it during their Monday on their own? Or would we end up with people skipping it do you think?

Reply:
I think we put it in the schedule only to make sure it gets completed. It could easily be part of the Friday or Monday activities. Many cohorts have asked for it to come before the weekend. But I would be surprised if this didn't add a bit of chasing to the CF's workload.